Assets/HoloToolkit/Sharing/Scripts/VoiceChat/MicrophoneTransmitter.cs

```diff
@@ -271,9 +271,9 @@ private void SendFixedSizedChunk(NetworkConnection connection, byte[] data, int
             msg.Write(cameraPosRelativeToGlobalAnchor.z);
 
             // HRTF direction bits
```

Comment:
Is there a reason we are leaving this in commented out, vs just deleting the section?

Reply:
Because potentially someone could implement the hrtf parts.

Reply:
But the current implementation is wrong?  Sorry, i don't know much about this stuff.

Reply:
Yeah it's causing a mismatch in the stream in the receiving end.

Reply:
Got it.  Seems like maybe we should just put a TODO in on the Transmitter like in the Receiver.  Or put a comment in on why that code is commented out.

Personally I'm not a big fan of leaving commented code in.

Reply:
I could comment it back in on the receiving end too, but I figured this way was saving network traffic.